test/AtomicFileTest.cpp

```diff
@@ -150,16 +151,47 @@ int fiu_test (UnitTest& t)
 
 int fiu_test (UnitTest& t)
 {
-  t.skip ("AtomicFile::write_raw throws on error");
-  t.skip ("AtomicFile::finalize_all() throws on error");
-  t.skip ("AtomicFile::reset clears failure state");
-  t.skip ("AtomicFile::append throws on error");
-  t.skip ("AtomicFile::append did not partially fill the file.");
-  t.skip ("AtomicFile::append failures prevent finalization");
+  t.skip ("AtomicFileTest: AtomicFile::write_raw throws on error");
+  t.skip ("AtomicFileTest: AtomicFile::finalize_all() throws on error");
+  t.skip ("AtomicFileTest: AtomicFile::reset clears failure state");
+  t.skip ("AtomicFileTest: AtomicFile::append throws on error");
+  t.skip ("AtomicFileTest: AtomicFile::append did not partially fill the file.");
+  t.skip ("AtomicFileTest: AtomicFile::append failures prevent finalization");
   return 0;
 }
 #endif // FIU_ENABLE
 
+//////////////////////////////////////////////////////////////////////////////
+int test_symlink (UnitTest& t)
+{
+  Path link ("link");
+  Path target ("target");
+  File(target).write_raw("empty\n");
+
+  const std::string write_contents = "test file contents\n";
+
+  if (::symlink (target._data.c_str(), link._data.c_str()) != 0)
+  {
+    throw std::runtime_error("Failed to create link in test_symlink");
+  }
+
+  {
+    AtomicFile file(link);
+    file.write_raw(write_contents);
+  }
+  AtomicFile::finalize_all ();
+
+
+  // After finalizing we should be able read the data that we wrote through the
+  // target
+  std::string read_contents;
+  AtomicFile::read (target, read_contents);
+  t.is(read_contents, write_contents, "AtomicFile writes to symlinks end up in target");
+  t.is (link.is_link (), true, "AtomicFile to link shall maintain symlink");
```

Comment:
Isn't the group missing here, compared to the other tests? 🤔 
```suggestion
  t.is(read_contents, write_contents, "AtomicFileTest: AtomicFile writes to symlinks end up in target");
  t.is (link.is_link (), true, "AtomicFileTest: AtomicFile to link shall maintain symlink");
```

Reply:
Good catch, updated.